test/components/BrowserDetector.test.tsx

```diff
@@ -1,10 +1,10 @@
-import React from 'react';
-import renderer from 'react-test-renderer';
+import * as React from 'react';
+import * as renderer from 'react-test-renderer';
 
 import BrowserDetector from '../../src/BrowserDetector';
 
 function forceUserAgent(ua) {
-  navigator.__defineGetter__('userAgent', () => ua);
+  navigator['__defineGetter__']('userAgent', () => ua);
```

Comment:
@giogonzo `__defineGetter__` is this one https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/__defineGetter__

In theory it could be replaced by

```ts
Object.defineProperty(navigator, 'userAgent', { get: () => ua })
```

but it throws an error when calling it multiple times, like I'm doing in these tests. One possible solution is something like


```ts
let _ua = '';
Object.defineProperty(navigator, 'userAgent', { get: () => _ua })
```

and then just change the `_ua` value, but I didn't like it and `__defineGetter__` works as expected, so... ¯\_(ツ)_/¯